src/components/Navigation/TopBar/Resources/TransferToAppDialog.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+
+import {
+  Button,
+  Dialog,
+  Icon,
+  LinearProgress,
+  Typography,
+} from '@equinor/eds-core-react';
+import { close } from '@equinor/eds-icons';
+
+import AnimatedCheckmark from 'src/components/Feedback/AnimatedCheckmark';
+import { spacings } from 'src/style';
+import string from 'src/utils/string';
+
+import styled from 'styled-components';
+
+const PortalDialog = styled(Dialog)`
+  width: 400px;
+  height: 323px;
+`;
+
+const Container = styled.div`
+  gap: ${spacings.medium};
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  max-width: 370px;
+  padding: ${spacings.medium} ${spacings.large};
+`;
+const StyledTypography = styled(Typography)`
+  span {
+    text-transform: capitalize;
+  }
+`;
+
+const Loading = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacings.large};
+`;
+
+const TransferringContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacings.small};
+  width: 100%;
+  align-items: center;
+  padding: ${spacings.large} 0;
+`;
+
+interface PortalTransitProps {
+  onClose: () => void;
+  applicationName: string;
+  url: string;
+}
+
+const TransferToAppDialog: FC<PortalTransitProps> = ({
+  onClose,
+
+  applicationName,
+  url,
+}) => {
+  const [finished, setFinished] = useState(false);
+  const [progress, setProgress] = useState(0);
```

Comment:
If you want you can use the new `useFakeProgress` hook here 👀 
Something link this:
```suggestion
const { finished, progress } = useFakeProgress({ onDone: () => window.open(url, '_self') });
```

Reply:
OMG , you fixed the hook ? :O amazing, it worked 👌